packages/itwinui-css/backstop/tests/tile.html

```diff
@@ -280,7 +290,7 @@ <h2>Everything turned on</h2>
           </div>
         </div>
 
-        <div class="iui-tile-content ">
+        <div class="iui-tile-content">
```

Comment:
Is this an avoidable change? Because when I tried to format the file after readding the deleted space, the formatter kept it as is.

Or maybe it is good to clean up spaces and cause these small changes in the PR, in this case

Reply:
I've reverted the empty space cleanup for easier reviewing.